Assets/Scripts/Globals.cs

```diff
@@ -41,15 +41,17 @@ public static Vector3 DirectTargeting(Transform from, Transform to)
     {
         { EnemyType.TURRET, "Turret" },
         { EnemyType.HEALER, "Healer" },
-        { EnemyType.GRENADIER, "Grenadier"}
+        { EnemyType.GRENADIER, "Grenadier"},
+		{ EnemyType.RANGED, "Ranged"}
```

Comment:
extra space before the {RANGED, "Ranged"}

Reply:
spacing!!!